app/src/main/kotlin/co/touchlab/droidconandroid/EventDetailFragment.kt

```diff
@@ -33,11 +33,12 @@ class EventDetailFragment : Fragment(), EventDetailHost {
 
     private val viewModel: EventDetailViewModel by lazy {
         val helper = DatabaseHelper.getInstance(activity)
+        val oldDbHelper = DatabaseHelper.getInstance(activity)
```

Comment:
Whoops, that was left in when I was trying to have the old stuff and the new stuff side by side.